tests/test_s1_orbits.py

```diff
@@ -0,0 +1,37 @@
+import tempfile
+import requests
+import responses
+import pytest
+from pathlib import Path
+
+import s1_orbits
+
+
+@responses.activate
+def test_fetch_for_scene():
+    granule = "foo"
+    filename = "filename.txt"
+    request_url = f"{s1_orbits.API_URL}/{granule}"
+    response_url = f"https://bar.com/{filename}"
+    directory = tempfile.gettempdir()
```

Comment:
pytest provides the built-in `tmp_path` fixture for this purpose: https://docs.pytest.org/en/6.2.x/tmpdir.html